.github/workflows/publish.yml

```diff
@@ -8,25 +8,41 @@ on:
 
 jobs:
   build:
+    name: Package wheel
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.12"]
     steps:
       - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
-      - name: Set up Python ${{ matrix.python-version }}
+      - name: Set up Python
         uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: "3.13.0"
       - name: Install publish dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install '.[publish]'
+          python -m pip install --upgrade pip build
       - name: Build and test wheel
         run: |
           rm -rf dist/
           python -m build --sdist --wheel .
-          twine check dist/*
-      - name: Publish
-        run: |
-          twine upload  --non-interactive --verbose --username __token__ --password ${{ secrets.PYPI_API_TOKEN }} dist/*
+      - name: Store wheel
+        uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882  # v4.4.3
+        with:
+          name: beku-stackabletech-package
+          path: dist/
+  pypi-publish:
```

Comment:
question: I'm curious why we split this up into two different jobs? Couldn't we publish the package to PyPi in the same job which would save us the artifact up/download?

Reply:
Alright, makes sense. Thank you!